tests/phpunit/tests/rest-api/rest-block-directory-controller.php

```diff
@@ -202,6 +202,70 @@ public function test_get_item_schema() {
 		$this->assertArrayHasKey( 'humanized_updated', $properties );
 	}
 
+	/**
+	 * @ticket 53621
+	 */
+	public function test_get_items_response_conforms_to_schema() {
+		wp_set_current_user( self::$admin_id );
+		$plugin = $this->get_mock_plugin();
+		$cb     = function () use ( $plugin ) {
+			return (object) array(
+				'info'    =>
+					array(
+						'page'    => 1,
+						'pages'   => 1,
+						'results' => 1,
+					),
+				'plugins' => array(
+					$plugin,
+				),
+			);
+		};
+		add_filter( 'plugins_api', $cb );
+
+		$request = new WP_REST_Request( 'GET', '/wp/v2/block-directory/search' );
+		$request->set_query_params( array( 'term' => 'cache' ) );
+
+		$expected = array(
+			array(
+				'name'                => 'sortabrilliant/guidepost',
+				'title'               => 'Guidepost',
+				'description'         => 'A guidepost gives you directions. It lets you know where you’re going. It gives you a preview of what’s to come.',
+				'id'                  => 'guidepost',
+				'rating'              => 4.3,
+				'rating_count'        => 90,
+				'active_installs'     => 100,
+				'author_block_rating' => 0,
+				'author_block_count'  => 1,
+				'author'              => 'sorta brilliant',
+				'icon'                => 'https://ps.w.org/guidepost/assets/icon-128x128.jpg?rev=2235512',
+				'last_updated'        => gmdate( 'Y-m-d\TH:i:s', strtotime( $plugin['last_updated'] ) ),
+				'humanized_updated'   => sprintf( '%s ago', human_time_diff( strtotime( $plugin['last_updated'] ) ) ),
+				'_links'              => array(
+					'wp:install-plugin' => array(
+						array(
+							'href' => 'http://example.org/index.php?rest_route=%2Fwp%2Fv2%2Fplugins&slug=guidepost',
+						),
+					),
+					'curies'            => array(
+						array(
+							'name'      => 'wp',
+							'href'      => 'https://api.w.org/{rel}',
+							'templated' => true,
+						),
+					),
+				),
```

Comment:
Let's unset the `_links` field so we don't need to worry about comparing correct URLs.